packages/editor/.gitignore

```diff
@@ -8,6 +8,8 @@ pnpm-debug.log*
 lerna-debug.log*
 
 node_modules
+public/svg
```

Comment:
Okay to deploy the example source files, but svgs shouldn't be in the same folder anymore, so this shouldn't be needed.

Reply:
Removed.